apps/atlas/src/app/search/details/field/attributes-cards/attributes-cards.component.html

```diff
@@ -1,5 +1,5 @@
 <models4insight-details-cards-list
-  title="Data Attributes"
+  title="Business Terms"

```

Comment:
See other comment. This could for example be:

```suggestion
  [attr.title]="attributes | translate"
```

The pipe symbol applies the transformation using the translate operator. You might still need to import the `TranslateModule`, but you will find out if there's an error 😆